modules/alias.py

```diff
@@ -0,0 +1,108 @@
+import click
+import dev
```

Comment:
This line seems to be causing [build failure for Python 3](https://travis-ci.org/yoda-pa/yoda/jobs/365879074#L1412). It seems unused to me, so I think just removing this should fix it.

Reply:
I've made the changes along with a couple others. I'm still getting a build failure for Python 3, do you mind helping me understand where the error is coming from?